Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/v1beta3/conversion_generated.go

```diff
@@ -2053,7 +2053,22 @@ func convert_api_SecurityContextConstraints_To_v1beta3_SecurityContextConstraint
 	} else {
 		out.AllowedCapabilities = nil
 	}
-	out.AllowHostDirVolumePlugin = in.AllowHostDirVolumePlugin
+
+	if in.Volumes != nil {
+		out.Volumes = make([]FSType, len(in.Volumes))
+		for i, v := range in.Volumes {
+			out.Volumes[i] = FSType(in.Volumes[i])
+
+			// set the Allow* fields based on the existence in the volume slice
```

Comment:
ah yes, I just need to remove this completely if the v1beta3 api won't have this field.  It didn't complain because os/kube doesn't generate for that version.